src/index.js

```diff
@@ -20,9 +20,13 @@ module.exports = {
     })
   },
   getGlobalYarnParser: function(yarnPath, cb) {
+    console.log('yarnPath', yarnPath);
     // <yarn>/bin/yarn ==> (src) <yarn>/src/lockfile
     //                     (compiled) <yar>lib/node_modules/yarn/lib/lockfile/parse.js
-    var parser = require(path.resolve(yarnPath, '../../lib/lockfile/parse.js'))
+    var parserPath = path.resolve(yarnPath, '../../lib/lockfile/parse.js')
+    console.log('parserPath', parserPath)
+    var parser = require(parserPath)
+    console.log('parser', parser);
     return cb(null, parser.default)
```

Comment:
i dont actually see any change here.  still debugging windows?  sorry i didnt test windows, it wasnt available as a test target in gitlab.  i have used appveyor before, but didnt expect trouble in this tiny module